pkgs/tools/cd-dvd/nrg2iso/default.nix

```diff
@@ -0,0 +1,23 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  name = "nrg2iso-${version}";
+  version = "0.4";
+
+  src = fetchurl {
+    url = "gregory.kokanosky.free.fr/v4/linux/${name}.tar.gz";
+    sha256 = "18sam7yy50rbfhjixwd7wx7kmfn1x1y5j80vwfxi5v408s39s115";
+  };
+
+  installPhase = ''
+    mkdir -pv $out/bin/
+    cp -v nrg2iso $out/bin/nrg2iso
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A linux utils for converting CD (or DVD) image generated by Nero Burning Rom to ISO format";
+    homepage = http://gregory.kokanosky.free.fr/v4/linux/nrg2iso.en.html;
+    license = licenses.gpl2;
+    platforms = platforms.linux;
```

Comment:
I'm sorry but I needed this once and will probably forget about this in a month. It has 148 lines of code and hasn't been updated in more than a decade, so there's not much to maintain here.